README.md

```diff
@@ -45,6 +45,7 @@ Please, see [this guide](CONTRIBUTING.md), if you want to contribute to this lis
 - [IT KPI Rust](https://t.me/itkpi_rust) - **Telegram** - Чат ІТ KPI про Rust.
 - [IT KPI – Python](https://t.me/itkpi_python) - **Telegram** - Чат IT KPI обговорень про всякі штуки зі світу пайтону.
 - [PHP Friends Club Ukraine Chat](https://t.me/phpfriendsclub_chat) - **Telegram** - Чат про PHP та Web розробку.
+- [UAGolang](https://www.meetup.com/uagolang/) - **Website** - Ukrainian Golang User Groups
```

Comment:
```suggestion
- [UAGolang](https://www.meetup.com/uagolang/) - **Website** - Ukrainian Golang User Groups.
```

`awesome_lint` is unhappy if there is no dot.
Is this Meetup group in Kyiv? If so, probably it makes sense to put it in the `by location . kyiv` section.